### PR TITLE
E2E Testing add Prettier and Eslint

### LINKED ISSUE #1764: E2E Testing: Apply Formatting and Linting
- Prettier
- EsLint
- CI Task to check it

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to get e2e tests under formatting and eslint control.

Looks like a large PR, but it is 99% formatting alignment.

Closes #1764 

# Discussion
- Basic Prettier
- Eslint standards
- Eslint Mocha plugin
- Running lint in CI

### CODE REVIEW

File: .github/workflows/verify-pr-commit.yml [hunk 1]
```diff
@@ -694,6 +694,9 @@ jobs:
       - name: Install NPM Modules
         run: npm ci
         working-directory: e2e
+      - name: Lint
+        run: npm run lint
+        working-directory: e2e
```
Comment (wilwade) [inline .github/workflows/verify-pr-commit.yml:699]: Added to CI.

Sadly it is hard to run this separate from the e2e tests due to needing the api-augment package.

File: e2e/.mocharc.json
Comment (wilwade) [inline e2e/.mocharc.json:1]: Formatting only

File: e2e/.relay-chain.mocharc.json
Comment (wilwade) [inline e2e/.relay-chain.mocharc.json:1]: Formatting only

File: e2e/capacity/replenishment.test.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import "@frequency-chain/api-augment";
```
Comment (wilwade) [inline e2e/capacity/replenishment.test.ts:1]: All these test files are matching the formatting rules. No other changes.

File: e2e/package.json
Comment (wilwade) [inline e2e/package.json:1]: Tagging the changes here in the `package.json` file, but nothing unexpected.

File: e2e/tsconfig.json
Comment (wilwade) [inline e2e/tsconfig.json:1]: Formatting only.

**Review by JoeCap08055 — APPROVED** [review]
💯

File: e2e/capacity/replenishment.test.ts [hunk 3]
```diff
@@ -77,15 +76,17 @@ describe("Capacity Replenishment Testing: ", function () {
...
-      call.payWithCapacity('current'), { name: "RpcError", message: /1010.+account balance too low/ });
+    return assert.rejects(call.payWithCapacity('current'), {
+      name: 'RpcError',
+      message: /1010.+account balance too low/,
+    });
   }
 
-  describe("Capacity is not replenished", function () {
-    it("if out of capacity and last_replenished_at is <= current epoch", async function () {
-      let [stakeKeys, stakeProviderId] = await createAndStakeProvider("NoSend", 150n * CENTS);
```
Comment (enddynayn) [inline e2e/capacity/replenishment.test.ts:86]: how did we "let" this happen

**Review by enddynayn — APPROVED** [review]
much needed! 💯

**Pull request merged**